assets/js/googlesitekit/notifications/components/layout/SubtleNotification.js

```diff
@@ -30,11 +30,13 @@ import WarningSVG from '../../../../../svg/icons/warning.svg';
 import { Grid, Cell, Row } from '../../../../material-components';
 
 export default function SubtleNotification( {
+	className,
 	title,
 	description,
 	dismissCTA,
 	additionalCTA,
 	type = 'success',
+	icon = null,
```

Comment:
These new props should be added to `propTypes`.

(Also, why set `icon` to `null` instead of just using `undefined`?)

Reply:
Yeah not sure why included null, removed it now